test/server/lib/HostedStorageManager.ts

```diff
@@ -962,6 +963,97 @@ describe('HostedStorageManager', function() {
       });
     });
   }
+
+  describe('minio-without-redis', async () => {
+    const workerId = 'dw17';
+    let tmpDir: string;
+    let oldEnv: EnvironmentSnapshot;
+    let docWorkerMap: IDocWorkerMap;
+    let externalStorageCreate: ExternalStorageCreator;
+    let defaultParams: ConstructorParameters<typeof HostedStorageManager>;
+
+    before(async function() {
+      tmpDir = await createTmpDir();
+      oldEnv = new EnvironmentSnapshot();
+      // Disable Redis
+      delete process.env.REDIS_URL;
+
+      const creator = create?.getStorageOptions?.('minio')?.create;
+      if (!creator) {
+        return this.skip();
```

Comment:
Hmm this won't skip if minio is not configured. What do you think about something like:
```
      const storage = create?.getStorageOptions?.('minio');
      const creator = storage?.create;
      if (!creator || !storage?.check?.()) {
```